jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererCodeCoverageTracker.kt

```diff
@@ -91,6 +91,9 @@ abstract class CodeWhispererCodeCoverageTracker(
             LOG.debug { "event with isWholeTextReplaced flag: $event" }
             if (event.oldTimeStamp == 0L) return
         }
+
+        // We don't update totalToken if the document offset diff is larger than the threshold
+        if (event.newLength - event.oldLength >= DOCUMENT_CHANGE_THRESHOLD) return
```

Comment:
What's the reason behind this change?

Reply:
the main reason I think is to block that loading scenario https://telemetry-externalprod.ide-toolkits.dev-tools.aws.dev/_plugin/kibana/goto/7205999dd5c2f77543c4d0bf82e5d370?security_tenant=global

since we can still see cases like, 
 totalToken = 10000, acceptedToken = 0
 totalToken = 5000, acceptedToken = 0

Reply:
I suspect ^^ cases are a result of loading case.

Reply:
Also I am thinking to  add a field "isCodeWhispererAutoSuggestion: boolean" in the `codewhisperer_percentage` metric as well, it will help us read the metric easier, know if the 0% is reasonable or not, IMO.

Reply:
actually Lei already have set a threshold for VSC of length 20, but I would prefer larger number like hundred or thoushand

Reply:
don't x-link to internal resources please

Reply:
ackk